### PR TITLE
Deploy test toolshed under gunicorn

### PR DESCRIPTION
**Pull request by mvdbeek**
So we can run 22.05.

@natefoo that's how I deployed it now and it seems to be working.

### CODE REVIEW

File: env/testtoolshed/group_vars/toolshedservers/vars.yml [hunk 1]
```diff
@@ -17,29 +17,15 @@ toolshed_dbname: galaxy_testtoolshed
...
-    # FIXME: this breaks zerg mode
-    #stats: "127.0.0.1:9191"
-    processes: 1
-    threads: 2
-    offload-threads: 2
-    reload-on-rss: 1400
-    master: True
-    logto: "{{ galaxy_toolshed_log_dir }}/uwsgi.log"
-    pythonpath: lib
-    unique-cron: "0 -1 -1 -1 -1 {{ galaxy_venv_dir }}/bin/python scripts/tool_shed/build_ts_whoosh_index.py -c {{ galaxy_toolshed_config_file }} --config-section tool_shed -d"
```
Comment (mvdbeek) [inline env/testtoolshed/group_vars/toolshedservers/vars.yml:42]: This should probably go in a regular crontab ?

**Review by natefoo — APPROVED** [review]
I think we can just revert the changes to the supervisor template, then this won't affect the MTS (although the cron job will, so we could drop that from the `galaxy_toolshed_config` for the MTS?

Other than that, we don't do a `systemctl daemon-reload` or handle restarts after modifying the configs or service unit, but we probably didn't do a `supervisorctl update` or appropriate restarts before, so this is ok.

**Comment by mvdbeek** [comment]
We should also do https://gist.github.com/mvdbeek/81eaf567e832c3e2f971cd04955c7769, I think that would fix the shed diff issue

**Comment by natefoo** [comment]
@mvdbeek Do you recall why we run a standalone hgweb? Is the plan to remove it from the internal TS? Currently it isn't proxied and doesn't do anything, but I can configure it as such if that's the direction we're going.

**Comment by mvdbeek** [comment]
It's much faster, and hegen leaks memory like crazy.

**Comment by natefoo** [comment]
Cool, I proxied it over top `/repos` on the TTS although the TTS does appear to still run it internally. Also uses a lot of memory on startup while loading repos, is that why?

File: env/testtoolshed/group_vars/toolshedservers/vars.yml [hunk 2]
```diff
@@ -17,29 +17,23 @@ toolshed_dbname: galaxy_testtoolshed
 ## used by: other vars and various templates
 galaxy_toolshed_instance_codename: test
 galaxy_toolshed_instance_hostname: testtoolshed.g2.bx.psu.edu
-
+galaxy_toolshed_gunicorn_workers: 2
+galaxy_toolshed_gunicorn_bind: "127.0.0.1:9009"
+galaxy_toolshed_gunicorn_max_requests: 50000
+galaxy_toolshed_gunicorn_max_requests_jitter: 5000
```
Comment (mvdbeek) [inline env/testtoolshed/group_vars/toolshedservers/vars.yml:23]: We should probably decrease that if we're running standalone hgweb. That's a good way to end up with entries in the database but not within the hgweb config.

**Comment by mvdbeek** [comment]
The TS interface (in particular pagination) now works, that's so nice :)

**Pull request merged**